src/app/bios.rs

```diff
@@ -512,6 +512,10 @@ impl Component for BiosComponent {
                 }
                 println!();
             }
+
+            // Invalidate the CMOS checksum to force writing the defaults
+            let mut cmos = cmos::Cmos::default();
+            cmos.write(123, 0);
```

Comment:
I think both bytes should be written 0. It is a 16-bit value.

Reply:
It now writes `old+1` for both the hi and lo bytes to ensure it's different.